userdocs/src/usage/gitops-v2.md

```diff
@@ -0,0 +1,111 @@
+# New: GitOps with Flux v2
+
+!!! Warning "Flux v2 `eksctl` integration is Experimental"
+    Partial experimental support for [Flux v2 (part of the GitOps Toolkit)](https://toolkit.fluxcd.io/)
+    is available from `eksctl` version `0.38.0`. See [below](#experimental-installing-flux-v2-gitops-toolkit),
+    with significant API changes likely until development has stabilised.
+    All disruptions will be noted in [releases](https://github.com/weaveworks/eksctl/releases).
+
+[Gitops][gitops] is a way to do Kubernetes application delivery. It
+works by using Git as a single source of truth for Kubernetes resources
+and everything else. With Git at the center of your delivery pipelines,
+you and your team can make pull requests to accelerate and simplify
+application deployments and operations tasks to Kubernetes.
+
+[gitops]: https://www.weave.works/technologies/gitops/
+
+## Installing Flux v2 (GitOps Toolkit)
+
+_Initial Flux v2 support was released in `0.38.0`, but the API has undergone significant changes. An older
+example can be found [here](https://github.com/weaveworks/eksctl/blob/a854bdbd3e47059d860a78600c2c1813281f4ebf/examples/12-gitops-toolkit.yaml)._
+
+From version `0.50.0`, `eksctl` provides an option to alternatively bootstrap [Flux v2](https://toolkit.fluxcd.io/)
```

Comment:
update this version if https://github.com/weaveworks/eksctl/pull/3662 ends up in a later version